packages/execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -152,8 +176,29 @@ export class BaseSnapExecutor {
       return;
     }
 
+    const methodObject =
+      EXECUTION_ENVIRONMENT_METHODS[
+        method as keyof typeof EXECUTION_ENVIRONMENT_METHODS
+      ];
+
     // support params by-name and by-position
-    const paramsAsArray = sortParamKeys(methodObject, params);
+    const paramsAsArray = sortParamKeys(methodObject.params, params);
```

Comment:
I wonder if we should just support one of these - doesn't have to be for this PR though.